packages/tools/src/tools/segmentation/strategies/utils/getStrategyData.ts

```diff
@@ -40,7 +40,10 @@ function getStrategyData({ operationData, viewport }) {
     // of that actor at that moment so we have the imageData already
     const actor = viewport.getActor(segmentationRepresentationUID);
     segmentationImageData = actor.actor.getMapper().getInputData();
-    const currentSegmentationImageId = imageIdReferenceMap.get(currentImageId);
+    const currentSegmentationImageIds = imageIdReferenceMap.get(currentImageId);
+
+    // Todo: fix this to support multiple derived imageIds
+    const currentSegmentationImageId = [...currentSegmentationImageIds][0];
```

Comment:
Leaving this code everywhere without having real implementations seems like a bad idea for putting this in core.  I'd rather see the old code here and then actually fix it once we understand what is actually going to happen.